Verse/src/Schemas/JSON/Reader.cs

```diff
@@ -377,8 +377,20 @@ private bool ReadToObjectFromArray<TObject>(ReaderState state, ILookup<int, Read
 				// Read object value
 				state.PullIgnored();
 
-				if (!(field.HasValue ? field.Value(this, state, ref target) : this.Skip(state)))
-					return false;
+				// Test null case

```

Comment:
I'm not sure I understand exactly what this change is doing, can you please explain?

Reply:
Hello,
Let's say that I am expecting an array and I get a Json like:
`
{
    myArray : null
}
`
Basically I want the decoder to behave as if I got an empty Json: 
`{ }`

In order to do it, we must return before the `field.Value(this, state, ref target)` is called preventing it to run the constructor and create the entity (witch should not since it is nulll).